modules/ags/src/main/scala/lucuma/ags/GuideStarCandidate.scala

```diff
@@ -35,13 +38,26 @@ final case class GuideStarCandidate(
 ) {
   def name: NonEmptyString =
     refineV[NonEmpty](s"Gaia DR3 $id").getOrElse(sys.error("Cannot happen"))
+
+  // Reset the candidate to a given instant
+  // This can be used to calculate and cache the location base on proper motion
+  // The tracking variables are reset to avoid double calculations
+  def at(i: Instant): GuideStarCandidate =
+    copy(tracking =
+      tracking.at(i).fold(tracking)(c => SiderealTracking.const(c).copy(epoch = tracking.epoch))
+    )
```

Comment:
Is the intention to zero out the PM information?

Reply:
```
gsc.at(someInstant).at(originalInstant) =/= gsc
```

Reply:
I guess we shouldn't I had an idea but actually what I probably want is only to reset the epoch to match the `Instant`